libs/zzz/formula/src/data/disc/disc.test.ts

```diff
@@ -201,12 +201,17 @@ describe('Disc sheets test', () => {
     expect(calc.compute(anby.final.dmg_.fire).val).toBeCloseTo(0.1) // 2p passive
     expect(calc.compute(anby.final.crit_).val).toBeCloseTo(0.05 + 0.28)
   })
-  it('InfernoMetal', () => {
```

Comment:
Removed a test by accident?

Reply:
test was duplicated above, this was cleanup + addition